src/debug/clean-pending-payments-for-dead-node.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * how to run:
+ *	. ./.envrc && yarn ts-node \
+ *		--files \
+ *			-r tsconfig-paths/register \
+ *			-r src/services/tracing.ts \
+ *		src/debug/clean-pending-payments-for-dead-node.ts
+ */
+
+import { LedgerTransactionType, UnknownLedgerError } from "@domain/ledger"
+
+import { isUp } from "@services/lnd/health"
+import { params as unauthParams } from "@services/lnd/unauth"
+
+import { setupMongoConnection } from "@services/mongodb"
+import { LedgerService, translateToLedgerTx } from "@services/ledger"
+import { LockService } from "@services/lock"
+import { baseLogger } from "@services/logger"
+
+import { MainBook } from "@services/ledger/books"
+
+const DEAD_LND_PUBKEYS = [
+  "02ae3c066e67cf3951bb5230cf4e13aee053bc06465cd25f870988b691469140b0", // dead node in hack environment
```

Comment:
those should be passed from the cli?

Reply:
we may not need an array. I think it's ok to run the cli once for every node that may needs some clean up (currently running 2 would means we have removed both node which would itself be unlikely)

Reply:
Changed: https://github.com/GaloyMoney/galoy/pull/1583/commits/1c39f876c162ab2a5c726bc76bf6ea85d2527a3d